### PR TITLE
html forms

### PR DESCRIPTION
**Pull request by IvanTkachenko3301**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://IvanTkachenko3301.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://IvanTkachenko3301.github.io/layout_html-form/report/html_report/)
```
Comment (liliia-livandovskaya) [inline readme.md:4]: Don't forget to add Demo and Test links here:

Screenshot 2023-02-20 at 11 59 59

File: src/index.html [hunk 2]
```diff
@@ -3,15 +3,144 @@
...
+          >
+        </label>
+  
+        <label class="age">
+          How old are You?
+          <input type="number"
+          name="age" min="1"
+          max="100" value="12"
+          required
+          >
```
Comment (liliia-livandovskaya) [inline src/index.html:44]: [checklist p.3 ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

fix it all over the page

File: src/style.css [hunk 3]
```diff
@@ -1 +1,72 @@
...
+  margin-bottom: 20px;
+}
+
+.label-last-form {
+  display: block;
+}
+
+.comments {
+  margin-bottom: 10px;
+}
```
Comment (liliia-livandovskaya) [inline src/style.css:72]: there is no need to duplicate styles for each input.  Rewrite the style to simplify it

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Good job! Pay attention to the comments and checklist

File: src/index.html (hunk 2 above)
Comment (IvanTkachenko3301) [reply src/index.html:44]: done

**Pull request closed without merging**